open_city_profile/tests/snapshots/snap_test_graphql_api_schema.py

```diff
@@ -243,7 +255,7 @@
   updateMyProfile(input: UpdateMyProfileMutationInput!): UpdateMyProfileMutationPayload
   updateProfile(input: UpdateProfileMutationInput!): UpdateProfileMutationPayload
   deleteMyProfile(input: DeleteMyProfileMutationInput!): DeleteMyProfileMutationPayload
-  deleteMyServiceData(input: DeleteMyServiceDataMutationInput!): DeleteMyServiceDataMutationPayload
+  deleteMyServiceData(input: DeleteMyServiceDataInput): DeleteMyServiceDataMutation
```

Comment:
The input argument should be required, i.e. there should be an `!` in the end of the type.